src/test/java/com/puppycrawl/tools/checkstyle/CheckerTest.java

```diff
@@ -777,6 +779,8 @@ public void testCatchErrorInProcessFilesMethod() throws Exception {
              * Test is checking catch clause when exception is thrown.
              *
              * @noinspection ProhibitedExceptionThrown
+             * @noinspectionreason ProhibitedExceptionThrown - There is no other way to obey
+             *      haltOnException field.
```

Comment:
This and the others do not appear to be testing haltOnException...

Reply:
Yes, you are right. Done.